docs/_config.yml

```diff
@@ -1,4 +1,5 @@
 remote_theme: just-the-docs/just-the-docs
```

Comment:
as far as I understand you replaced the remote github theme to a local theme. Do we still need the "remote_theme" here then?

Reply:
I think we need both? I have the theme in the `gem` install for local, and for GitHub we can specify the remote one.

Reply:
I'm not sure about this, I didn't find in Github docs how to use a local theme. It says it supports either a "supported theme" or the one hosted on HitHub with `remote-theme`. https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/adding-a-theme-to-your-github-pages-site-using-jekyll#adding-a-theme
So adding both might break Github deployment. We can try redeploying docs from your branch.

Reply:
But this won't work because it's a fork. I guess we can try merging and see what happens.

Reply:
I'll fast follow if it breaks.